### ISSUE TITLE
Should look for  1build.yaml in parent and fetch the closest one

### ISSUE THREAD

**Issue by AdeshAtole** [gopinath-langote/1build#91]
## Description
Some codebases have build commands run from sub directories as well as from main project dir. Can be managed with multiple `1build.yaml`s , but good to have a single one across project.

**Comment by landpro** [comment]
@AdeshAtole what if commands are supposed to run from the working directory? should the tool automatically switch to the working directory and run the command?

We gave it a thought and decided to hold this one until the demand is more clear.

In the meantime, I would suggest solving your requirement with a symlink – should work if the number of subfolders is not large.

We appreciate your proactiveness and the PR – but please make sure that the issue is discussed before implementing it – revisit our Contribution page for more details.

https://github.com/gopinath-langote/1build/blob/master/CONTRIBUTING.md

**Comment by landpro** [comment]
@AdeshAtole we are looking forward to hearing more feedback about your subfolder issue and if symlinks work for you.

In the meantime feel free to pick up an issue with a `help wanted` label from our backlog – https://github.com/gopinath-langote/1build/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22

Just updated the contributing page to clarify the procedure.

**Comment by AdeshAtole** [comment]
@landpro Yes, symlinks should suffice me for now. The idea was to have 1build analogous to `git`, where git works anywhere in the subtree.

> what if commands are supposed to run from the working directory?  should the tool automatically switch to the working directory and run the command?

This question isn't clear to me. 1build commands always run in current working directory, right?

It does make sense to keep this on hold till things get finalised.
